core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %}
+
+{% block maintitle %}
+  Oppia Editor
+{% endblock maintitle %}
+
+{% block subtitle %}
+  {{ title }}
+{% endblock subtitle %}
+
+
+{% block header_js %}
+  {{ super() }}
+  <script type="text/javascript">
+    GLOBALS.can_edit = JSON.parse('{{can_edit|js_string}}');
+  </script>
+
+  <style>
+    html, body {
+      background-color: #eee;
+    }
+  </style>
+
+  {{dependencies_html}}
+{% endblock header_js %}
+
+
+{% block content %}
+  <div ng-controller="CollectionEditor">
+	  <p>title: <[collection.title]></p>
```

Comment:
Should all of these ``<p>`` tags be wrapped in an ng-if for whether the collection has been loaded yet? Otherwise, the initial editor page will show 'title: ', etc. with empty fields until the collection is loaded. It's mostly up to you, I just thought I would suggest it/open it as a discussion.